haiku-apps/haikuwebsearch/haikuwebsearch-0.0.0~git.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="A search application"
+DESCRIPTION="A Haiku Application that provides a replicant to search the internet."
+HOMEPAGE="https://github.com/Oyami-Srk/HaikuWebSearch"
+COPYRIGHT="2017 Shiroko"
+LICENSE="MIT"
+REVISION="1"
+commit="c9bc31b35f066870cc73bca78c3f02ac331b1d3f"
+SOURCE_URI="$HOMEPAGE/archive/$commit.tar.gz"
+CHECKSUM_SHA256="86ccf3e816fac123ac156bb54678001506ddb59e292c3b2a3f4ea36fd9719de1"
+SOURCE_FILENAME="HaikuWebSearch-$portVersion-$commit.tar.gz"
+SOURCE_DIR="HaikuWebSearch-$commit"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+
+PROVIDES="
+	HaikuWebSearch = $portVersion
+	app:Haiku_WebSearch = $portVersion
+	"
+
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:gcc
+	cmd:make
+	"
+
+USER_SETTING_FILES="
+	settings/HaikuWebSearch directory
+	settings/HaikuWebSearch/engines
+	"
```

Comment:
HaikuWebSearch only [looks](https://github.com/Oyami-Srk/HaikuWebSearch/blob/master/SearchView.cpp#L96) for engines in `B_USER_SETTINGS_DIRECTORY`. This means `USER_SETTING_FILES` (actually it's `USER_SETTINGS_FILES`) is not needed as this only applies to files within `B_SYSTEM_SETTINGS_DIRECTORY`